docs/src/modules/java/pages/running-locally.adoc

```diff
@@ -125,3 +125,73 @@ include::java:example$event-sourced-customer-registry-subscriber/src/main/resour
 ----
 
 With both services configured, we can start them independently by running `mvn compile exec:java` in two separate terminals.
+
+[#local_cluster]
+== Running a local cluster
+
+For testing clustering behavior and high availability scenarios, you can run your Akka service as a local cluster with multiple nodes. This allows you to simulate a distributed environment on your local machine.
+
+=== Database requirement
+
+To run in cluster mode, you need a shared database that all nodes can connect to. The `local-nodeX.conf` files configure the application to connect to a PostgreSQL database.
+
+Before starting your cluster nodes, you must start a local PostgreSQL database. We recommend using Docker Compose for this purpose. Create a `docker-compose.yml` file with the following configuration:
+
+[source, yaml]
+----
+services:
+  postgres-db:
+    image: postgres:17
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-d", "postgres", "-U", "postgres"]
+      interval: 5s
+      retries: 5
+      start_period: 5s
+      timeout: 5s
+----
```

Comment:
We don't document how to run locally with a real database only with a on file H2.

Now that we are showing how to form a local cluster, we could re-evaluate if it's not better to simply ship a docker-compose.yml with the samples and show how to use run with a postgres.